src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -111,6 +111,12 @@ internal static partial class Crypto
             int second,
             [MarshalAs(UnmanagedType.Bool)] bool isDst);
 
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int CheckX509Hostname(SafeX509Handle x509, string hostname, int cchHostname);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int CheckX509Hostname(IntPtr x509, string hostname, int cchHostname);
```

Comment:
I've now added the call to X509_dup to promote the IntPtr to a SafeHandle, and added the using; so deleted the IntPtr version here.